source/javascripts/all.js

```diff
@@ -17,22 +17,28 @@ $(document).ready(function() {
             var c = content.substr(0, showChar);
             var h = content.substr(showChar, content.length - showChar);
 
-            var html = c + '<span class="moreellipses">' + ellipsestext+ '&nbsp;</span><span class="morecontent"><span>' + h + '</span>&nbsp;&nbsp;<a href="" class="morelink">' + moretext + '</a></span>';
-
+            // var html = c + '<span class="moreellipses">' + ellipsestext+ '&nbsp;</span><span class="morecontent"><span>' + h + '</span>&nbsp;&nbsp;<a href="" class="morelink">' + moretext + '</a></span>';
+            var html = c + '<span class="moreellipses">' + ellipsestext+ '&nbsp;</span><span class="morecontent"><span>' + h + '</span>&nbsp;&nbsp;<a class="morelink">' + moretext + '</a></span>';
             $(this).html(html);
         }
     });
 
-    $(".morelink").click(function(){
-        if($(this).hasClass("less")) {
-            $(this).removeClass("less");
-            $(this).html(moretext);
-        } else {
-            $(this).addClass("less");
-            $(this).html(lesstext);
-        }
-        $(this).parent().prev().toggle(400);
-        $(this).prev().toggle();
-        return false;
-    });
+    // $(".morelink").click(function(){
+        
+        // if($(this).hasClass("less")) {
+        //     $(this).removeClass("less");
+        //     $(this).html(moretext);
+        // } else {
+        //     $(this).addClass("less");
+        //     $(this).html(lesstext);
+        // }
+        // $(this).parent().prev().toggle(400);
+        // $(this).prev().toggle();
+        // return false;
+    // });
 });
+
+
+
+
```

Comment:
clean up trailing whitespace so there is only one new line at the end of the file